recipes/restinio/v0.7/conanfile.py

```diff
@@ -36,13 +36,24 @@ def layout(self):
 
     def requirements(self):
         self.requires("llhttp/9.1.3")
-        self.requires("fmt/10.2.1")
-        self.requires("expected-lite/0.6.3")
+
+        if Version(self.version) >= "0.7.3":
+            self.requires("fmt/[>=11.0.2]")
+            self.requires("expected-lite/0.8.0")
+        else:
+            self.requires("fmt/10.2.1")
+            self.requires("expected-lite/0.6.3")
 
         if self.options.asio == "standalone":
-            self.requires("asio/1.29.0")
+            if Version(self.version) >= "0.7.4":
+                self.requires("asio/1.32.0")
+            else:
+                self.requires("asio/1.31.0")
         else:
-            self.requires("boost/1.84.0")
+            if Version(self.version) >= "0.7.4":
+                self.requires("boost/[>=1.87.0]")
```

Comment:
```suggestion
                self.requires("boost/1.87.0")
```

I don't recommend using version range for Boost because there is no guarantee about full compatibility between minors. Boost libraries can change their ABI compatibility (e.g. require a newer C++ standard) and the API (e.g. Boost process v2 become the default over v1)